### DISCUSSION TITLE
Instagram 將鼓勵青少年用戶「休息一下」

### DISCUSSION THREAD

**Discussion by 239216** [1101-GS4719-C/Python-Community-GS4719-C#79] [General]
Nick Clegg 承諾 Instagram 將會推出「Take a break」功能，鼓勵青少年停止使用社交網絡一段時間，推出的時間暫時未定

Nick Clegg 同時指出 Facebook 會「引導」青少年遠離 app 中「可能對他們沒有益處」的內容，但也確實提議需要對 Facebook 的演算法設立「問責」機制，包括在需要時可新增社群條例等，去確保符合現實世界的規定。其在將青少年引導至有害內容的同時，又只會刪除一小部份的仇恨言論。
Facebook 新推出的措施仍未畢能使 Haugen 及其他批評人士滿意。Facebook 新推出的功能雖然可以減少有害內容的曝光，但他們始終拒絕把有問題的內容刪除。
https://tw.news.yahoo.com/instagram-%E5%B0%87%E9%BC%93%E5%8B%B5%E9%9D%92%E5%B0%91%E5%B9%B4%E7%94%A8%E6%88%B6-%E4%BC%91%E6%81%AF-%E4%B8%8B-120015421.html

**Comment by HSU-YU-JEN** [comment]
最近因為Facebook與它旗下的Instagram、whatsapp、messenger大當機事件，有接觸到成癮與影響到青少年的新聞，希望社交平台可以正視這個問題，並提出有效的改善方法

**Comment by Jie2222** [comment]
鼓勵青少年休息的想法是好的，希望能夠有實際行動落實

**Comment by chupochun** [comment]
網路的便利有時候也會產生許多麻煩的問題

**Comment by bloodtrail0511** [comment]
確實!現在有超過一半的青少年平均每天使用手機3小時，是需要一個適當的機制控制大家的使用習慣

**Comment by 110409538** [comment]
不只是青少年,許多成年人也會沉迷於網絡世界,也許這項功能也適用於所有群體.

**Comment by 11timyyy** [comment]
有時真的需要外力來幫忙自己從網路中拔出來😂

**Comment by ghost** [comment]
我大臉書那麼愛廣告商的公司，也會有一天顧慮到社會大眾嗎...?

**Comment by lx3211** [comment]
可是青少年願意被管制嗎
所以之後ig會設置年齡身份檢驗嗎
會逃的還是會謊報年齡來躲避休息吧

**Comment by yucheng1028** [comment]
怕就是會跟中國一樣雖然有時間限制，但還是無法有效落實

**Comment by xkllkx** [comment]
我認為限制可能也會引起反效果，反而讓青少年用戶更想使用。

**Comment by d26488399** [comment]
我不喜歡被這種東西限制，雖然我很少滑IG。

**Comment by sean1325418** [comment]
不想休息的時候一定會有人找方法來破解這規定。真的想滑的人才不會被這麼輕易阻止

**Comment by yi-1111** [comment]
消彌沉迷是件刻不容緩的事，出發點立意良善

**Comment by Yuhsin000** [comment]
我覺得這應該是青少年父母與該青少年的責任，應該自己學習控管使用IG等社群媒體的時間、有閱聽人的素質，而非企業的責任。

**Comment by dayu1004** [comment]
我記得後台好像有使用時間的紀錄 如果真的使用太久再限制就好了 不然如果是通勤的時候被限制滑社群軟體的話也太痛苦了吧qwq

**Comment by hapiwangy** [comment]
instergram在確實是十分的方便，但實際上也很容易浪費很多的時間。通過第三方來限制年輕人的使用時間確實是有必要的

**Comment by yuhao112** [comment]
推出這項確實能幫助到我，畢竟自己常常在社交軟體上浪費很多不必要得時間。

**Comment by etin2468** [comment]
限制時間這個設計也要看是什麼的限制，如果聊天訊息都鎖的話反彈應該會很大？

**Comment by henrywen547** [comment]
雖然這個做法可能有助於減低離線焦慮症患者，但卻可能讓一般使用者也受到影響，實行之前還是要好好想清楚!

**Comment by Jay-722** [comment]
我真的需要限制一下，但其實可以自己鎖起來

**Comment by Lai27330** [comment]
感覺如果是靠著外力強迫休息的話 反而會讓真正沈迷的人更想鑽漏洞 而且重要訊息傳過來的時間無法預測 若臨時有要事卻收到時間限制的影響 可能會造成其他的麻煩

**Comment by HuanYuuuuuu** [comment]
我記得Youtube也可以設定休息一下的功能
但我通常看到她提醒我休息一下我就換關掉 然後繼續看..

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，希望未來大家不要太沉迷於網絡世界。

**Comment by minniewei** [comment]
現在我也常常覺得自己花太多時間在社群媒體上了

**Comment by Rosancl** [comment]
不曉得所謂Take a break的詳細內容會是怎樣？會不會影響到訊息功能，要是會的話倒是一個麻煩的問題。

**Comment by Jay-722** [comment]
我需要take a break ，滑太久

**Comment by jason79461385** [comment]
我的都被NBA洗版:D 到後來都懶得滑了

**Comment by treeann** [comment]
適當的休息一夏八~~~

**Comment by alicia0725** [comment]
沒記錯的話蘋果手機也有類似的功能，只不過是可以自己設置休息時間，或許ig也用這種模式較不會照成青少年反彈不願意使用。我也建議大家遠離一些不必要的社交，可以讓自己稍微休息一下

**Comment by zephyrxin01** [comment]
我覺得設計提醒的立意良好，也許也可以適時讓演算法推薦用眼過度的警示科普資訊，也許能更有效的讓使用者意識到休息的重要性

**Comment by Miinteaa** [comment]
雖然take a break 功能的設計出發是好的，但我認為要使用手機多長時間應該由自己判斷並調整，不然想滑手機的人還是會過度使用，無法根除壞習慣。

**Comment by harrycoconut** [comment]
這個take a break紀錄感覺會跟apple內建的app using time limit很像，沒辦法控制自己的人到最後也還是會取消這個設定，若真的要休息，建議刪掉instagram比較好喔😌

**Comment by Jamesmoneya** [comment]
這樣的設計感覺表面上只是要符合某些規定讓小孩可以適當休息，但是我覺得實質上完全沒有用，連父母當面告訴小孩不要看他們都不見得會聽了，那些想看的人，怎麼可能因為一個通知或警告就停止，就是做做樣子的

**Comment by ALAN30359704** [comment]
我認為使用手機的Instagram App頻率過高，並不是Instagram該負責的事情，而是使用者的問題，所以我們應該要去解決的是這些使用者如何去減少使用頻率，而非檢討App，但由於青少年自制力較薄弱，所以應由家長或老師加以提醒！

**Comment by sjh501** [comment]
讓使用者能夠更長時間的使用、更高頻率的瀏覽，本來就是這些企業或者應用程式的目標，因為這樣不論是討論度、廣告費甚至軟體內的付費收入都會增加。所以看到這個報導我只會覺得很好笑，那些抗議的人雖然初衷很好但不切實際，也不會認為那些發布聲明的企業會真的執行，切斷自己的財路。

**Comment by Chiuhsianglin** [comment]
手機成癮真的很恐怖，有時候早上起來滑個手機，看個短影片就會一直看下去，看時間才發現已經過去半小時或一個小時了，我認為app提醒「休息一下」的效果可能很微小，重點是用戶本身自己的克制

**Comment by alstinlin** [comment]
take a break對於正在滑ig的時候應該起不到甚麼效果，尤其是看影片看到一半的時候，幾乎不太可能停下來，所以花再滑ig上的時間長短還是取學於使用者，或者可以幫take a break的功能加上一個鬧鐘，至少可以讓使用者跳出ig頁面~

**Comment by pudding0803** [comment]
這個問題其實滿酷的，因為他們如果推出這個功能，應該會間接減少廣告的曝光率，也就會減少收入，但如果他們願意為了人們的健康著想，我是非常支持的！不過有沒有效果又是另一個問題了🤧

**Comment by roma0323** [comment]
跟各位分享一個實用的辦法，我自己在段考周+-1個星期都會把IG 或是FB的app刪掉，改用網頁版登入。
有兩個好處:
1.不會有通知來,讓你不容易突然呼其來的通知打斷
2.網頁版非常難用，會讓你很懶得滑下去(尤其是Reels)
基於以上兩個點，讓你不容易被吸引，就算被吸引也不容易深陷其中，大幅減少使用時間

**Comment by ashleyy-huang** [comment]
感覺現在有許多青少年都只活在網路之中，時不時就一直低頭滑手機，一滑也就是好幾個小時，導致其生產率下降許多，且我一直認為IG現在的reels跟Youtube的Shorts真的是害人不淺，浪費了好多時間

**Comment by larry40305** [comment]
雖然加上些許限制對網路成癮有幫助，但演算法必須要再改進。因為他常常會推給我奇怪的東西。

**Comment by bestbanana295** [comment]
我覺得這種東西對於管制青少年使用IG的效果有限，畢竟青少年連父母的話都不一定會聽了怎麼還會裡一個IG的用量提渻呢？

**Comment by zhiron544** [comment]
文章讓我們了解 Instagram 的新功能和 Facebook 的處境。我認為這個功能是一個好的嘗試，但也要考慮到使用者的自主性和需求。我希望 Facebook 能夠更加負責任地管理其平台，保護青少年和社會的利益。

**Comment by ThanatosJun** [comment]
訊息輛快速流通，社交媒體扮演重要的一環，而就是因此，我們對於各種大小來自手機的訊息都極為關注，因此時不時拿起手機，深怕漏掉重要的訊息，因此有了手機成癮症。若要解決這個問題，首先要改變的很可能是溝通的方式。
